packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -230,38 +221,44 @@ export default {
         };
     },
 
+    computed: {
+        ...mapState('fAccountInfoModule', [
+            'consumer'
+        ])
+    },
+
     watch: {
         isAuthFinished: {
             immediate: true,
-            handler (value) {
+            async handler (value) {
                 if (value) {
-                    this.initialise();
+                    await this.initialise();
                 }
             }
         }
     },
 
-    mounted () {
+    created () {
+        if (!this.$store.hasModule('fAccountInfoModule')) {
+            this.$store.registerModule('fAccountInfoModule', fAccountInfoModule);
+        }
+    },
```

Comment:
Don't we normally do this in the consuming application? At least we are doing that for Checkout.

Reply:
Yes, this is now the preferred way ... (_before it was thought before that we couldn't do this and used to do this in the parent (coreweb)_)